apps/web/app/api/cron/weekly_summary/route.ts

```diff
@@ -180,7 +180,7 @@ const getProducts = async (): Promise<ProductData[]> => {
                   },
                 },
                 select: {
-                  status: true,
+                  responseId: true,
```

Comment:
@ShubhamPalriwala only the select statement changed here. So is the value even used? Or are there more changes needed?

Reply:
Ahh, ok, think I got it :-)

Reply:
I think it still breaks for existing Displays that don't have the responseId and is not able to calculate the response rate correctly, right?

Reply:
i've discussed with @Dhruwang and updated this field with the always-present `id` field. Please take a look now